deploy/apecloud-mysql/templates/_helpers.tpl

```diff
@@ -60,3 +60,96 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Agamotto config
+*/}}
+{{- define "agamotto.config" -}}
+extensions:
+  memory_ballast:
+    size_mib: 32
+
+receivers:
+  apecloudmysql:
+    endpoint: ${env:ENDPOINT}
+    username: ${env:MYSQL_USER}
+    password: ${env:MYSQL_PASSWORD}
+    allow_native_passwords: true
+    database:
+    collection_interval: 15s
+    transport: tcp
+  filelog/error:
+    include: [/data/mysql/log/mysqld-error.log]
+    include_file_name: false
+    start_at: beginning
+  filelog/slow:
+    include: [/data/mysql/log/mysqld-slowquery.log]
+    include_file_name: false
+    start_at: beginning
+
+processors:
+  memory_limiter:
+    limit_mib: 128
+    spike_limit_mib: 32
+    check_interval: 10s
+
+exporters:
+  prometheus:
+    endpoint: 0.0.0.0:{{ .Values.metrics.service.port }}
+    send_timestamps: false
+    metric_expiration: 20s
+    enable_open_metrics: false
+    resource_to_telemetry_conversion:
+      enabled: true
+  apecloudfile/error:
+    path: /var/log/kubeblocks/${env:KB_NAMESPACE}_${env:DB_TYPE}_${env:KB_CLUSTER_NAME}/${env:KB_POD_NAME}/error.log
+    format: raw
+    rotation:
+      max_megabytes: 10
+      max_days: 3
+      max_backups: 1
+      localtime: true
+  apecloudfile/slow:
+    path: /var/log/kubeblocks/${env:KB_NAMESPACE}_${env:DB_TYPE}_${env:KB_CLUSTER_NAME}/${env:KB_POD_NAME}/slow.log
+    format: raw
+    rotation:
+      max_megabytes: 10
+      max_days: 3
+      max_backups: 1
+      localtime: true
+
+service:
+  telemetry:
+    logs:
+      level: info
+  extensions: [ memory_ballast ]
+  pipelines:
+    metrics:
+      receivers: [ apecloudmysql ]
+      processors: [ memory_limiter ]
+      exporters: [ prometheus ]
+    logs/error:
+      receivers: [filelog/error]
+      exporters: [apecloudfile/error]
+    logs/slow:
+      receivers: [filelog/slow]
+      exporters: [apecloudfile/slow]
+{{- end }}
+
+{{/*
+Agamotto config for proxy
+*/}}
+{{- define "proxy-monitor.config" -}}
```

Comment:
able to provider a common helper function that is provided in `kblib` chart?

Reply:
Now, `kblib` only used by cluster helm chart, if this function is used by other cluster helm chart, it is suitable to add it to `kblib`.